### PR TITLE
SMU: update to new v3 .dat files and first part of geometry sensing

### PR DESCRIPTION
**Pull request by pfrstg**
Switch parsing and writing to the latest .dat inputs
Significant updates to the documentation in dataset.proto
Add output mode of original atomic input files
Add make_sqlite using the new TFRecord output
Add utilities from the geometry sensing, but it's not yet hooked up into the pipeline.
   (note that there is a bunch of Julia and C++ code, but we don't need that to actually run)

### CODE REVIEW

**Comment by pfrstg** [comment]
@googlebot I fixed it.

File: smu/dataset_pb2.pyi [hunk 1]
```diff
@@ -0,0 +1,1373 @@
+"""
```
Comment (kjk33) [inline smu/dataset_pb2.pyi:1]: Did you intentionally include this generated file?

Comment (pfrstg) [reply smu/dataset_pb2.pyi]: No, thanks for the catch!

**Comment by pfrstg** [comment]
All Julia code has been removed. I also had to put in a couple of unittest fixes (and hacks).
I will get the other author to respond.

**Comment by IanAWatson** [comment]
I have submitted a CLA, that should be OK now, not sure why there are ongoing failures.

**Comment by IanAWatson** [comment]
@googlebot I consent.

**Review by kjk33 — APPROVED** [review]

**Pull request merged**